script/webpack-client.js

```diff
@@ -0,0 +1,40 @@
+const webpack = require("webpack");
+const webpackDevServer = require('webpack-dev-server');
+const path = require('path');
+const baseConfig = require('../config/webpack.config.dist');
+
+const port = 8080;
+const serverURI = `webpack-dev-server/client?http://localhost:${port}/`;
+
+let browser = baseConfig();
+browser.output.libraryTarget = "var";
+browser.output.filename = "./dist/seng-boilerplate.js";
+browser.output.path = path.join(__dirname, '../dist');
+
+if(typeof browser.entry === 'string')
+{
+	const { entry }  = browser;
+	browser.entry = { app: [serverURI, entry]};
```

Comment:
You don't need to make entry an object with an array, entry can be an array itself.